src/astro/sql/__init__.py

```diff
@@ -1,10 +1,16 @@
 from typing import Callable, Dict, Iterable, List, Mapping, Optional, Union
 
 import pandas as pd
-from airflow.decorators.base import TaskDecorator, task_decorator_factory
+
+try:
+    from airflow.decorators.base import TaskDecorator, task_decorator_factory
+except Exception:
```

Comment:
@dimberman it would be great to use a specific exception here (e.g. `ImportError`)? To use `except Exception` could hide other errors and make it tricky to debug.